HW10/10-02-outside/BuildingSpheres.py

```diff
@@ -0,0 +1,24 @@
+import math
+
+class Sphere(object):
+    def __init__(self, radius, mass):
```

Comment:
It's not a good idea to define volume, surface area and density in the constructor. If we change any attributes (radius or mass), we'll have inconsistency with those three attributes. Therefore, we should return volume, surface area and density only from methods